wordpress-mobile/AztecEditor-iOS
# Issue/487 fix emoji

Fixes #487 

To test:
 - Start the empty demo 
 - Insert a single emoji
 - Check it doesn't crash
